packages/components/src/components/form/SelectBar/SelectBar.tsx

```diff
@@ -216,7 +217,7 @@ export const SelectBar: <V extends ValueTypes>(props: SelectBarProps<V>) => JSX.
                 {options.map(option => (
                     <Option
                         key={String(option.value)}
-                        onClick={handleOptionClick(option)}
+                        onClick={handleOptionClick(isDisabled, option)}
```

Comment:
I think `isDisabled` dont need to be passed as argument, it is already available in the handler from component props.

Reply:
I think I had a good reason for this. But I'll take a look again. Now seems that `isDisabled` really doesn't need to be there. Thanks!